docs/orchestrating-elastic-stack-applications/elasticsearch/transport-settings.asciidoc

```diff
@@ -111,12 +113,13 @@ spec:
             driver: csi.cert-manager.io
             readOnly: true
             volumeAttributes:
-              csi.cert-manager.io/issuer-name: ca-cluster-issuer
+              csi.cert-manager.io/issuer-name: ca-cluster-issuer <2>
               csi.cert-manager.io/issuer-kind: ClusterIssuer
               csi.cert-manager.io/dns-names: "${POD_NAME}.${POD_NAMESPACE}.svc.cluster.local"
 ----
+<1> Disables the self-signed certificates generated by ECK for the transport layer.
 
-The example assumes that a `ClusterIssuer` by the name of `ca-cluster-issuer` exists and a PEM encoded version of the CA certificate is available in a ConfigMap (in the example named `trust`).  The CA certificate must be in a file called  `ca.crt` inside the ConfigMap in the same namespace as the Elasticsearch resource.
+<2> The example assumes that a `ClusterIssuer` by the name of `ca-cluster-issuer` exists and a PEM encoded version of the CA certificate is available in a ConfigMap (in the example named `trust`).  The CA certificate must be in a file called `ca.crt` inside the ConfigMap in the same namespace as the Elasticsearch resource.
 
```

Comment:
I think the below `Certificate.spec.issuerRef.name` needs to be updated to `ca-cluster-issuer`, not `selfsigned-issuer`.

Reply:
So the idea was that we have a self-signed issuer which issues a certificate (`root-ca-secret`) which is then used in secod issuer the `ca-cluster-issuer` to work around the restriction of the CSI driver that it cannot work with self-signed issuers